src/components/design-system/Input.tsx

```diff
@@ -0,0 +1,443 @@
+import React, {
+  useEffect,
+  useCallback,
+  useRef,
+  useState,
+  forwardRef,
+  ReactNode,
+  ComponentProps,
+  MutableRefObject,
+} from 'react';
+import { styled, css } from '@storybook/theming';
+import { WithTooltip, TooltipMessage } from '@storybook/components';
```

Comment:
This was changed from 

```ts
import { WithTooltip, TooltipMessage } from '@storybook/design-system';
```

and I do not know whether there is a big difference, whether this will break or not. Please test it well

Reply:
This may have had a minor impact: https://www.chromatic.com/review?appId=6480e1b0042842f149cfd74c&number=171&type=linked&view=changes